### DISCUSSION TITLE
Support textboxes without [type='text']

### DISCUSSION THREAD

**Discussion by bandleader** [tailwindlabs/tailwindcss#9188] [Ideas]
In your README you write

> Note that for text inputs, you must add the type="text" attribute for these styles to take effect. This is a necessary trade-off to avoid relying on the overly greedy input selector and unintentionally styling elements we don't have solutions for yet, like input[type="range"] for example.

Why not simply target both `input[type=text]` and `input:not[type]`? See https://jsfiddle.net/4guvpjyk/

**Comment by krtko1** [comment]
I agree that inputs without type should have same style as type text, because type text is default type. This also make better compatibility with mimify-html as discussed in https://github.com/wilsonzlin/minify-html/issues/95
